### PR TITLE
Invite users to group

### PR DESCRIPTION
**Pull request by igaponov**
https://github.com/PowerlineApp/powerline-mobile/issues/116

### CODE REVIEW

File: backend/src/Civix/CoreBundle/Service/Group/GroupManager.php [hunk 1]
```diff
@@ -270,4 +272,24 @@ public function changePermissionSettings(Group $group)
...
     }
+
+    /**
+     * @param Group $group
+     * @param User $inviter
+     * @param string|array $userNames
+     */
+    public function joinUsersByUsername(Group $group, User $inviter, $userNames)
+    {
+        $users = $this->entityManager->getRepository(User::class)->findBy(['username' => $userNames]);
```
Comment (austinpapp) [inline backend/src/Civix/CoreBundle/Service/Group/GroupManager.php]: what if the user is already a member? are we handling that?

Comment (igaponov) [inline backend/src/Civix/CoreBundle/Service/Group/GroupManager.php]: Fixed!

**Comment by austinpapp** [comment]
@igaponov 

```
1) Civix\ApiBundle\Tests\Controller\BookmarkControllerTest::testIndexAction
Failed asserting that two arrays are equal.
```

see https://travis-ci.org/PowerlineApp/powerline-server/builds/146575709#L587

**Comment by igaponov** [comment]
Done

**Pull request merged**